src/mockchain/lib/Convex/MockChain.hs

```diff
@@ -225,41 +227,6 @@ legacyPlutusArgsToData = \case
   LegacyPlutusArgs2 redeemer scriptContext -> [redeemer, PV3.toData scriptContext]
   LegacyPlutusArgs3 datum redeemer scriptContext -> [datum, redeemer, PV3.toData scriptContext]
 
-
```

Comment:
These types were moved to Convex.Class to avoid circular dependencies between mockchain and base modules.